docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -405,7 +405,11 @@ CI Testing
       * ``validate-modules:nonexistent-parameter-documented``
       * ``validate-modules:parameter-list-no-elements``
       * ``validate-modules:parameter-type-not-in-doc``
-      * ``validate-modules:undocumented-parameter``
+
+  * The following validations MUST not be ignored except in specific circumstances:
+      * ``validate-modules:undocumented-parameter``: this MUST only be ignored in one of these two cases:
+        1. A dangerous module parameter has been deprecated or removed, and code is present to inform the user that they should not use this specific parameter anymore or that it stopped working intentionally.
+        2. Module parameters are only used to pass in data from an accompanying action plugin.
```

Comment:
It doesn't seem the nested ordered list is formatted correctly:

![Rendered](https://github.com/ansible/ansible-documentation/assets/9920591/a6822851-791d-4112-a405-45dfe3b4c1e9)